### DISCUSSION TITLE
How to Hide or Remove Picklist  move to all source and move to all target  buttons

### DISCUSSION THREAD

**Discussion by dssenan** [primefaces/community#679] [PrimeNG] [answered]
How to Hide or Remove Picklist  move to all source and move to all target  buttons

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, you cannot remove move to all buttons, there isn't a feature for it. You can only hide/remove source and target control buttons by `showSourceControls` and `showTargetControls` inputs. If you don't need to move items between lists, you can use OrderList  component instead.
